### ISSUE TITLE
I can not get the text with accents

### ISSUE THREAD

**Issue by rcdiaz** [lapwinglabs/x-ray#144]
I'm trying to retrieve a text that has accents but these appear to me as?, then use iconv to convert Windows-1252 but all my characters with accents are replaced by ý.

My code is this:
exports.buscar = function (req, res){
var x = Xray();
x('http://www.google.com', 'div .x7', [{
    descripcion: '.tx',
    precio: '.x11 .pr'
}])
(function(err, obj) {
    var busqueda = JSON.stringify(obj);
    var utf8String = iconv.decode(busqueda, 'Windows-1252');
    res.send(utf8String);
});

**Comment by gnujeremie** [comment]
I'm using the [entities](https://www.npmjs.com/package/entities) package to decode my french characters. Maybe this could help you.

**Comment by Kikobeats** [comment]
@gnujeremie awesome! Do you think that could be sense integrate the package with the library?

**Comment by gnujeremie** [comment]
@Kikobeats Yes, I think it could be usefull, at least for many european people :D

**Comment by Kikobeats** [comment]
Yeah, I think that is a very tyical case and have sense handle it. 

@gconnolly do you think that you can convert this into a PR?

**Comment by gconnolly** [comment]
Sign me up.

**Comment by gconnolly** [comment]
Hmm... looking into how/where to integrate `entities`. I have an ideas, but I am struggling in setting up a reproducible case. @rcdiaz, could you provide some sample HTML or a URL to where you are seeing this issue?
